File: Bubble_sort.py
def bubble_sort(sequence):
    indexing_length = len(sequence) - 1
    done = False

    while not done:
        done = True
        for i in range(0, indexing_length):
            if sequence[i] > sequence[i + 1]:
                done = False
                sequence[i], sequence[i + 1] = sequence[i + 1], sequence[i]
    return sequence
print(bubble_sort([4, 9, 10, 6, 18, 20]))

File: BubbleSort.java
public class BubbleSort {
	
	public static void main(String[] args) {
		
		int[] integers = {10,6,12, 4, 100, 2, 18,9};
		for(int i=0; i< integers.length; i++) {
			for(int j=0; j<integers.length-i-1; j++) {
				int k = integers[j];
				if(k>integers[j+1]) {
					integers[j]= integers[j+1];
					integers[j+1] = k;
				}
			}
		}
		
		for(int i:integers) {
			System.out.println(i);
		}
	}

}
